lib/hydra/derivatives/processors/video/config.rb

```diff
@@ -61,5 +61,16 @@ def default_size_attributes
       def default_audio_attributes
         "-ac 2 -ab 96k -ar 44100"
       end
+
+      def default_audio_encoder
+        @audio_encoder_capability = Hydra::Derivatives::CapabilityService.new
+        @audio_encoder_capability.capture_output
+        audio_encoder = if @audio_encoder_capability.fdk_aac?
```

Comment:
Please push the audio_encoder decision into the object you created. As written the current class needs to know to ask if it's `fdk_aac?` but then interprets the result into the type of encoder. This logic makes more sense (to me) to be in the object you created.